include/crm/pengine/internal.h

```diff
@@ -49,16 +49,16 @@ pcmk_resource_t *pe__last_group_member(const pcmk_resource_t *group);
 #  define pe_proc_err(fmt...) { was_processing_error = TRUE; crm_err(fmt); }
 #  define pe_proc_warn(fmt...) { was_processing_warning = TRUE; crm_warn(fmt); }
 
-#define pe__set_working_set_flags(working_set, flags_to_set) do {           \
-        (working_set)->flags = pcmk__set_flags_as(__func__, __LINE__,       \
+#define pe__set_working_set_flags(scheduler, flags_to_set) do {             \
```

Comment:
We could rename these macros themselves since they're internal

Reply:
Yep, later PR